pkgs/development/python-modules/cypari2/default.nix

```diff
@@ -37,11 +37,12 @@ buildPythonPackage rec {
     export PYTHONPATH="$out/${python.sitePackages}:$PYTHONPATH"
 
     # install "." instead of "*.whl"
-    ${python.pythonForBuild.pkgs.bootstrapped-pip}/bin/pip install . --no-index --no-warn-script-location --prefix="$out" --no-cache
+    pip install . --no-index --no-warn-script-location --prefix="$out" --no-cache
   '';
 
   nativeBuildInputs = [
     pari
+    python.pythonForBuild.pkgs.pip
```

Comment:
Naive question: Just adding pip is not splicing correctly?

Reply:
You probably know more about this than I do. I don't yet know how the splicing is implemented, and I was not able to build up to `cypari2` with the cross build command I used. For now I left this as-is and only tested building cypari2 non-cross to limit the risk, but this is something I'd like to look into when I have time.

Reply:
The splicing will need to work. If it doesn't yet then we have a regression and it would be likely that the issue occurs in more cases.